src/callinput.c

```diff
@@ -210,6 +213,26 @@ char callinput(void)
 		break;
 	    }
 
+	case 17:	// CTRL+q
+	    {
+		if (qtcdirection & 1) {
+		    qtc_main_panel(RECV);
+		}
+		if (qtcdirection == 2) {
+		    qtc_main_panel(SEND);
+		}
```

Comment:
Ah, I see. It looks a little weird but should work. It would be good to add a comment about it or use an else between the two cases. THat way the next one will not stumble upon it.